addons/uav/XEH_postInit.sqf

```diff
@@ -0,0 +1,32 @@
+#include "script_component.hpp"
+
+["Air", "init", {
+    params ["_vehicle"];
+    if (!unitIsUAV _vehicle) exitWith {};
+    if (!hasPilotCamera _vehicle) exitWith {};
+    if ((getNumber (configFile >> "CfgVehicles" >> (typeOf _vehicle) >> QUOTE(ADDON) >> "trackPilotCamera")) == 1) then {
+        GVAR(singleOperatorUAVs) pushBack _vehicle;
+    };
+}, true, [], true] call CBA_fnc_addClassEventHandler;
+
+
+GVAR(pfehID) = [{
+    if (count GVAR(singleOperatorUAVs) < 1) exitWith {};
+    {
+        if(!alive _x) exitWith {
+            GVAR(singleOperatorUAVs) deleteAt (GVAR(singleOperatorUAVs) find _x);
+        };
+        private _pilotCameraTarget = getPilotCameraTarget _x;
+        if(!isUAVConnected _x && {!(_pilotCameraTarget select 0)}) exitWith {};
```

Comment:
```suggestion
        if (!isUAVConnected _x && {!(_pilotCameraTarget select 0)}) exitWith {};
```

Reply:
This will exit the `forEach GVAR(singleOperatorUAVs);` loop, which I don't think is intended behavior
(unrelated to style suggestion)

Reply:
This was written before the advent of the `break` and `continue` commands, and as such had these thinking `exitWith` behaved similarly with loops. Will correct.